### DISCUSSION TITLE
use nostr-relaypool-ts?

### DISCUSSION THREAD

**Discussion by PeerRich** [NostrGit/NostrGit#80] [General]
https://github.com/adamritter/nostr-relaypool-ts looks like a decent typescript package for connecting to nostr-relays

**Comment by reyamir** [comment]
in my experiences, [nostr-relaypool-ts](https://github.com/adamritter/nostr-relaypool-ts) provide more API and easy to control than other libraries, and it can support cache too.

**Comment by davidabram** [comment]
I am already using it in https://github.com/NostrGit/NostrGit/pull/32.
